pkg/cloudprovider/aws/packing/packing.go

```diff
@@ -60,11 +68,15 @@ func (p *podPacker) Pack(ctx context.Context, constraints *cloudprovider.Constra
 	// Sort pods in decreasing order by the amount of CPU requested, if
 	// CPU requested is equal compare memory requested.
 	sort.Sort(sort.Reverse(binpacking.ByResourcesRequested{SortablePods: constraints.Pods}))
-	packings := []*Packing{}
+	var packings []*Packing
 	var packing *Packing
+	var err error
 	remainingPods := constraints.Pods
 	for len(remainingPods) > 0 {
-		packing, remainingPods = p.packWithLargestPod(remainingPods, constraints)
+		packing, remainingPods, err = p.packWithLargestPod(remainingPods, constraints)
+		if err != nil {
+			return packings, err
```

Comment:
Talk to Nick about our error annotation patterns. He should be able to share some ideas on whether or not this should be annotated.

Reply:
I chatted w/ Nick and annotated this err